src/DustExtinction.jl

```diff
@@ -150,8 +149,9 @@ directory containing the two FITS files defining the map,
 the FITS files defining the map open, speeding up repeated queries
 for E(B-V) values.
 """
-type SFD98Map
-    mapdir::Compat.UTF8String
+mutable struct SFD98Map
+    #mapdir::Compat.UTF8String depreciated
```

Comment:
You can remove this line.  We don't need to keep old code, everything can be easily resurrected from git history, if needed :slightly_smiling_face: